src/components/form-input/form-input.component.jsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+import React from 'react';
+
+import { FormInputContainer } from './form-input.styles';
+
+const FormInput = ({ handleChange, label, ...otherPorps }) => (
```

Comment:
I guess you meant `otherProps`:
```suggestion
const FormInput = ({ handleChange, label, ...otherProps }) => (
```